src/layouts/AssetManager/components/AssetControlCenter/components/sections/VenueFilteringSection.tsx

```diff
@@ -108,6 +125,19 @@ export const VenueFilteringSection: React.FC<VenueFilteringSectionProps> = ({
         onClose={() => setComingSoonModalOpen(false)}
         feature={comingSoonFeature}
       />
+
+      <ConfirmationModal
+        isOpen={removeConfirmOpen}
+        onClose={() => {
+          setRemoveConfirmOpen(false);
+          setVenueIdToRemove(null);
+        }}
+        onConfirm={confirmRemoveVenue}
+        title="Remove Venue Restriction"
+        message="Are you sure you want to remove this venue from the allowed venues list? This will prevent this venue from being used for settlement instructions involving this asset."
```

Comment:
minor nit add the venue ID being deleted and remove the line "This will prevent this venue from being used for settlement instructions involving this asset." as that can depend on whether restrictions are enabled or not
```suggestion
        message={`Are you sure you want to remove venue ${venueIdToRemove} from the allowed venues list?`}
```